test/schema/partner_show.js

```diff
@@ -16,11 +17,17 @@ describe('PartnerShow type', () => {
       displayable: true,
     }));
 
+    total
+      .onCall(0).returns(Promise.resolve(42))
+      .onCall(1).returns(Promise.resolve(2));
+
     PartnerShow.__Rewire__('gravity', gravity);
+    PartnerShow.__Rewire__('total', total);
```

Comment:
@dzucconi I’d really like some input on this one, currently its tests are failing.